src/__tests__/patients/edit/EditPatient.test.tsx

```diff
@@ -185,6 +185,70 @@ describe('Edit Patient', () => {
     expect(wrapper.update.isInvalid === true)
   })
 
+  it('should pass invalid phone number error when input is not a number on save button click', async () => {
+    let wrapper: any
+    await act(async () => {
+      wrapper = await setup()
+    })
+
+    const generalInformationForm = wrapper.find(GeneralInformation)
+
+    act(() => {
+      generalInformationForm.prop('onFieldChange')('phoneNumber', 'Not a number')
+    })
+
+    wrapper.update()
+
+    const saveButton = wrapper.find(Button).at(0)
+    const onClick = saveButton.prop('onClick') as any
+    expect(saveButton.text().trim()).toEqual('actions.save')
+
+    await act(async () => {
+      await onClick()
+    })
+
+    wrapper.update()
+    expect(wrapper.find(GeneralInformation).prop('errorMessage')).toMatch(
+      'patient.errors.updatePatientError',
+    )
+    expect(wrapper.find(GeneralInformation).prop('feedbackFields').phoneNumber).toMatch(
+      'patient.errors.patientPhoneNumberFeedback',
+    )
+    expect(wrapper.update.isInvalid === true)
```

Comment:
this can be written as `expect(wrapper.prop('isInvalid)).toBeTruthy()` to more closely align to how other tests have been written